app/models/Actionlog.php

```diff
@@ -1,85 +1,193 @@
 
-    public function adminlog()
-    {
-        return $this->belongsTo('User','user_id')->withTrashed();
-    }
+        public function licenselog()
+        {
 
-    public function userlog()
-    {
-        return $this->belongsTo('User','checkedout_to')->withTrashed();
-    }
+            return $this->belongsTo( 'License', 'asset_id' )
+                        ->withTrashed();
+        }
 
-    public function childlogs()
-    {
+        public function accessorylog()
+        {
 
-        return $this->hasMany( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Accessory', 'accessory_id' )
+                        ->withTrashed();
+        }
 
-    public function parentlog()
-    {
+        public function consumablelog()
+        {
 
-        return $this->belongsTo( 'ActionLog', 'thread_id' );
-    }
+            return $this->belongsTo( 'Consumable', 'consumable_id' )
+                        ->withTrashed();
+        }
 
-    /**
-	* Check if the file exists, and if it does, force a download
-	**/
-    public function get_src() {
+        public function adminlog()
+        {
 
-			$file = app_path().'/private_uploads/'.$this->filename;
-			return $file;
+            return $this->belongsTo( 'User', 'user_id' )
+                        ->withTrashed();
+        }
 
-    }
+        public function userlog()
+        {
 
+            return $this->belongsTo( 'User', 'checkedout_to' )
+                        ->withTrashed();
+        }
 
+        public function childlogs()
+        {
 
-    /**
-    * Get the parent category name
-    */
-    public function logaction($actiontype)
-    {
-        $this->action_type = $actiontype;
+            return $this->hasMany( 'ActionLog', 'thread_id' );
+        }
 
-        if($this->save()) {
-            return true;
-        } else {
-            return false;
+        public function parentlog()
+        {
+
+            return $this->belongsTo( 'ActionLog', 'thread_id' );
+        }
+
+        /**
+         * Check if the file exists, and if it does, force a download
+         **/
+        public function get_src()
+        {
+
+            $file = app_path() . '/private_uploads/' . $this->filename;
+
+            return $file;
+
+        }
+
+        /**
+         * Get the parent category name
+         */
+        public function logaction( $actiontype )
+        {
+
+            $this->action_type = $actiontype;
+
+            if ($this->save()) {
+                return true;
+            } else {
+                return false;
+            }
         }
-    }
-}
+
+        /**
+         * getUnacceptedAssets
+         * @param array $listOfAssets
+         *
+         * @return mixed
+         * @static
+         * @author  Vincent Sposato <vincent.sposato@gmail.com>
+         * @version v1.0
+         */
+        public static function getUnacceptedAssets( $listOfAssets = [ '*' ] )
+        {
+
+            $assetIds = implode( ",", $listOfAssets );
```

Comment:
Hah, if I had a dollar for all the things I wish I could tell people not to do :D 

Make sure you do a pull first, as I had to change some code to account for the reports when there were no assets awaiting acceptance. 

Not sure why, but the build is still failing since the migration. :(

https://travis-ci.org/snipe/snipe-it/jobs/74141878 

```
1) ExampleTest::testSomethingIsTrue
Illuminate\Database\QueryException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'snipeit_laravel.asset_logs' doesn't exist (SQL: select * from `asset_logs` order by `asset_id` asc, `created_at` asc)
```

![644529_hayao-miyazaki-spirited-away-spirited-away-gif-studio-ghilbi-coal-soot-gremlins](https://cloud.githubusercontent.com/assets/197404/9074647/7e74515c-3ac0-11e5-9d49-da5164cf0751.gif)